templates/part.board.mainView.php

```diff
@@ -5,12 +5,12 @@
 		<p>{{ statusservice.text }}</p></div>
 </div>
 
-<div id="board-header">
-	<div id="app-navigation-toggle" class="icon-menu" ng-click="navibar.show=!navibar.show"></div>
-	<a class="crumb" href="#" title="<?php p($l->t('All Boards')); ?>">
-		<i class="icon icon-home"></i>
-		<span class="hidden-visually"><?php p($l->t('All Boards')); ?></span>
-	</a>
+<div id="controls">
```

Comment:
I remember we discussed that some times ago, yes, although I'm not sure what exactly the reason was for that. :see_no_evil: Let's use our fixed approach, we can always go to the way files does it once it is implemented properly there.